Spectrum_Peak_Visualization.py

```diff
@@ -0,0 +1,151 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import peakutils
+import SPEFile
+
+"""
+plot_spectrum plots the spectra only.
+"""
```

Comment:
Docstrings go inside the `def` statement. The docstring should describe what the function does, including inputs, outputs, and behavior.